cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -258,10 +258,10 @@ <h2>Methodology</h2>
                 {% endif %}
 
                 {% if conf_file.get("script") %}
-                    <a href="#" ng-controller="filePreviewModalCtrl as $fPrevModalCtrl"
+                    <!-- <a href="#" ng-controller="filePreviewModalCtrl as $fPrevModalCtrl"
```

Comment:
Same as the previous files box code, perhaps deletable already.

Reply:
As stated in the previous comment, I will leave it for now